database/database-orders.go

```diff
@@ -74,6 +78,11 @@ func (s *Service) CancelOrder(orderID uuid.UUID, currentDate time.Time) error {
 	return ErrOrderFilledOrCancelled
 }
 
+// UpdateOrder updates a order
```

Comment:
We should add a comment that this function is not safe to be called with user provided arguments (like `UpdateOrder` below)